src/controllers/createpdf.controller.ts

```diff
@@ -0,0 +1,92 @@
+import { Controller, Get, Res, Post, Body, Render, Param, Query} from '@nestjs/common';
+import {Response} from 'express';
+import {createUserData, createLogInfo,createRunConf} from '../dtos/createpdf.dto';
+import {CreatePdfService} from "../services/createpdf.service";
+import { Log } from '../entities/log.entity';
+import { getManager } from 'typeorm';
+import { Run } from '../entities/run.entity';
+import { ApiUseTags, ApiBearerAuth } from '@nestjs/swagger';
+
+@ApiUseTags('createpdf')
+@Controller('createpdf')
+export class CreatepdfController {
+  constructor(public createPdfService: CreatePdfService){};
+
+  @Get('getlog')
+  async getLogInfo(@Query() query: createLogInfo, @Res() res: Response): Promise<void>{
```

Comment:
The query object here is not used as a query, either replace the @Query() with @Body() or @Param().